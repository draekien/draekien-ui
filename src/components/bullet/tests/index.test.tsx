import * as React from 'react';
import { render } from '@testing-library/react';
import Bullet from '..';

describe('Bullet component', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <Bullet color="primary" fullWidth invert>
        bullet
      </Bullet>
    );

    expect(container).toMatchSnapshot();
  });

  test('should match snapshot with defaults', () => {
    const { container } = render(<Bullet>bullet</Bullet>);

    expect(container).toMatchSnapshot();
  });
});
