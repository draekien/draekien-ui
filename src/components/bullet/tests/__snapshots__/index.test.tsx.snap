// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bullet component should match snapshot 1`] = `
<div>
  <div
    class="css-19na3e0-Bullet"
  >
    bullet
  </div>
</div>
`;

exports[`Bullet component should match snapshot with defaults 1`] = `
<div>
  <div
    class="css-17bfpm0-Bullet"
  >
    bullet
  </div>
</div>
`;
